Dockerfile

```diff
@@ -65,10 +64,13 @@ RUN cp /opt/TangoService/Tango/deployment/config/nginx.conf /etc/nginx/nginx.con
 RUN cp /opt/TangoService/Tango/deployment/config/supervisord.conf /etc/supervisor/supervisord.conf
 RUN cp /opt/TangoService/Tango/deployment/config/redis.conf /etc/redis.conf
 
+# Add user
+RUN useradd -ms /bin/bash autolab
```

Comment:
Why was this necessary?

Reply:
if you dont create the autolab user the tango command later fails to move the feedback file as it uses su autolab

Reply:
Ill have to look into this one more in my testing. Not sure what changed to require this